libs/langgraph/langgraph/pregel/runner.py

```diff
@@ -434,7 +434,8 @@ def commit(
                 raise exception
             else:
                 # save error to checkpointer
-                self.put_writes()(task.id, [(ERROR, exception)])  # type: ignore[misc]
+                task.writes.append((ERROR, exception))
```

Comment:
otherwise even handled errors would still prevent superset from completing

Reply:
I think the theoretically unsafe access to the maybe null weak ref, in reality our code elsewhere guarantees this doesn't expire halfway